run_tests.py

```
@@ -73,21 +77,18 @@ def _is_color_supported() -> bool:
     else:
         try:
             # Detect color support
-            result = subprocess.check_output(
-                ["tput", "colors"], 
-                stderr=subprocess.DEVNULL,
-                text=True
-            )
+            result = subprocess.check_output(["tput", "colors"], stderr=subprocess.DEVNULL, text=True)
```

Ruff (0.16.0)

[error] Starting a process with a partial executable path

(S607)